examples/gno.land/r/gnoland/ghverify/contract.gno

```diff
@@ -0,0 +1,140 @@
+package ghverify
+
+import (
+	"std"
+
+	"gno.land/p/demo/avl"
+	"gno.land/p/demo/gnorkle/feed"
+	"gno.land/p/demo/gnorkle/feeds/static"
+	"gno.land/p/demo/gnorkle/gnorkle"
+	"gno.land/p/demo/gnorkle/message"
+)
+
+const (
+	// The agent should send this value if it has verified the github handle.
+	verifiedResult = "OK"
+)
+
+var (
+	ownerAddress = string(std.GetOrigCaller())
+	oracle       *gnorkle.Instance
+	postHandler  postGnorkleMessageHandler
+
+	handleToAddressMap = avl.NewTree()
+	addressToHandleMap = avl.NewTree()
+)
+
+func init() {
+	oracle = gnorkle.NewInstance()
+	oracle.AddToWhitelist("", []string{ownerAddress})
+}
+
+type postGnorkleMessageHandler struct{}
+
+// Handle does post processing after a message is ingested by the oracle feed. It extracts the value to realm
+// storage and removes the feed from the oracle.
+func (h postGnorkleMessageHandler) Handle(i *gnorkle.Instance, funcType message.FuncType, feed gnorkle.Feed) {
```

Comment:
if you expect this pattern to be common, consider having a `PostMessageHandlerFunc` (like `http.HandlerFunc`)

Reply:
Would it be helpful if this type is only used to implement the `PostMessageHandler` interface?  I think using an interface instead of a typed function will ultimately give more flexibility to the realm that owns the oracle.